src/utils/processes.ts

```diff
@@ -192,7 +192,7 @@ export const fetchAllArNSRecords = async ({
   do {
     const pageResult = await contract
       .getArNSRecords({ cursor, limit: pageSize })
-      .catch((e: any) => {
+      .catch((e: Error) => {
```

Comment:
prefer any here bc we want to use `e?.message` and `e?.stack` if available without worrying about casting